src/js/tech/flash.js

```diff
@@ -321,7 +321,7 @@ Flash['checkReady'] = function(tech){
 // Trigger events from the swf on the player
 Flash['onEvent'] = function(swfID, eventName){
   let player = Lib.el(swfID)['player'];
-  player.trigger(eventName);
+  player.tech.trigger(eventName);
```

Comment:
We should store a reference to the tech on the swf object, probably [after it's created](https://github.com/videojs/video.js/blob/e1e69941a13d444525ccdf494d740eb1a3285320/src/js/tech/flash.js#L104). Then we don't need to reference the player here.